### ISSUE TITLE
Ensure that PULFALight is WCAG 2.1 AA accessible

### ISSUE THREAD

**Issue by jrgriffiniii** [pulibrary/pulfalight#109]
Please see https://accessibility.princeton.edu/Policy-and-Guidelines/digital-accessibility-guidelines/web-accessibility-guidelines for Princeton guidelines on ensuring that web content is accessible (and linking to some automated testing tools for this).

**Comment by jrgriffiniii** [comment]
We can state that we are just striving towards compliance with this, but if we have digitized audio/video without captioning or transcripts, we would need to note this.

**Comment by jrgriffiniii** [comment]
This epic should ensure that the PULFALight interfaces meet the WCAG 2.1 criteria. As banners and alerts should be checked for color contrast in order to meet these criteria, a part of this process is going to be manual.

**Comment by jrgriffiniii** [comment]
There already exists an accessibility audit template available at https://docs.google.com/spreadsheets/d/1g-J06sJHhY9XV1woIscElEHZEWHlJK9e8TzhO2uY1hk/edit#gid=1992549741
